<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeroManagerPreferences</class>
 <widget class="QDialog" name="NeroManagerPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="options">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="shortcutHide">
       <property name="text">
        <string>Hide to System Tray when starting shortcuts (Nero Manager will unhide when shortcut ends)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="defaultPrefixStartBox">
       <item>
        <widget class="QCheckBox" name="defaultPrefixStart">
         <property name="text">
          <string>Use default prefix to launch executables when given as a parameter
(ex. When launching from a file manager)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="defaultPrefix">
         <property name="placeholderText">
          <string>Select Default Prefix</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NeroManagerPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NeroManagerPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
